### ISSUE TITLE
Add new parameter or change pattern to function like grok match

### ISSUE THREAD

**Issue by rljohnsn** [elasticsearch/logstash#2123]
Need to be able to control what field the multiline filter is applying its logic too. Similar to how grok match parameter works.

```
grok {
   match => [
        "desired_field_to_match","%{GREEDYDATA:stuff}"
   ]
}
```

**Comment by wiibaa** [comment]
@colinsurprenant can you confirm this will be covered by https://github.com/logstash-plugins/logstash-filter-multiline/pull/4 with the new `source` config

**Comment by colinsurprenant** [comment]
confirmed! thanks @wiibaa

**Comment by colinsurprenant** [comment]
Fixed by  logstash-plugins/logstash-filter-multiline#4 published as v0.1.4
